tools/named_enum/named_enum/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
This seems to be a re-implementation of [`strum`](https://github.com/Peternator7/strum). We use `strum` in chain/client, let's keep it using instead of our own implementation. It seems that you want to use `EnumVariantNames` and `AsRefStr`

Reply:
Should have asked first, anyway, it was a good learning exercise! Will use strum instead.